src/elements/content-sidebar/activity-feed/comment/components/BaseCommentMenu.js

```diff
@@ -73,6 +74,9 @@ export const BaseCommentMenu = ({
                         data-resin-target={ACTIVITY_TARGETS.COMMENT_OPTIONS_EDIT}
                         data-testid="unresolve-comment"
                         onClick={() => handleStatusUpdate(COMMENT_STATUS_OPEN)}
+                        onMouseDown={(event: any) => {
```

Comment:
Likely want to add types to `event` instead of `any`

Reply:
Op, good catch.